libdnf/comps/group/group.cpp

```diff
@@ -192,7 +214,9 @@ std::vector<Package> Group::get_packages() {
         return packages;
     }
 
-    Pool * pool = query->sack->comps.p_impl->get_pool();
+    libdnf::solv::Pool & spool = get_pool(query->sack->comps.get_base());
+    Pool * pool = *spool;
+
     // Use only the first (highest priority) solvable for package lists
     Solvable * solvable = pool_id2solvable(pool, group_ids[0].id);
```

Comment:
(applies to most of the pool related changes in this commit)

You can call the methods of the wrapper directly, you're not supposed to use the `Pool *` unless necessary (for e.g. the `FOR_REPOS` and similar macros).

E.g. for the `pool_id2solvable()` call above:
```C
Solvable * solvable = spool.id2solvable(group_ids[0].id);
```

And I've called it `pool` everywhere unless there needed to be `Pool * pool` for the aforementioned macros, so unless you want to change the convention, can you do it the same way?

Reply:
I completely forgot to change this while making the other changes :facepalm: 
I hope it's all fixed now...